File: w06/d03/Exercises/weatherman.rb

# david's cold
puts "What's the condition out side?"
condition = gets.chomp.downcase
puts "What's the temperature?"
temperature = gets.chomp.downcase
def weatherman(condition, temperature)
	if condition == "rainy"
		if temperature == "cold"
			puts "better stay inside"
		elsif temperature == "hot"
			puts "sounds like a gross combination"
		end
	end
end
weatherman(condition, temperature)

File: w06/d03/Exercises/weatherman.js
var con = process.argv[2];
var temp = process.argv[3];

function weatherman(con, temp)
{
	if (con === "rainy" && temp === "cold")
	{
		return "Stay inside";
	}
	else if (con === "rainy" && temp === "hot")
	{
		return "Sounds gross";
	}
}
console.log(weatherman(con,temp));